.github/workflows/rubyonrails.yml

```
@@ -0,0 +1,40 @@
+name: "Rails Tests CI"
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+jobs:
+  rspec-test:
+    name: RSpec
+    runs-on: ubuntu-20.04
+    services:
+      postgres:
+        image: postgres:11-alpine
+        ports:
+          - "5432:5432"
+        env:
+          POSTGRES_DB: rails_test
+          POSTGRES_USER: rails
+          POSTGRES_PASSWORD: password
+    env:
+      RAILS_ENV: test
+      DATABASE_URL: "postgres://rails:password@localhost:5432/rails_test"
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Install Ruby and gems
+        uses: ruby/setup-ruby@8f312efe1262fb463d906e9bf040319394c18d3e # v1.92
+        with:
+          bundler-cache: true
+      - name: Assets precompile
+        run: |
+          bin/rails assets:precompile
+      - name: Run tests
+        run: bundler exec rspec
+      - name: Security audit dependencies
+        run: bundler exec bundler-audit --update
+      - name: Security audit application code
+        run: bundler exec brakeman -q -w2
+      - name: Lint Ruby files
+        run: bundler exec rubocop --parallel
```

I think Rails guide suggests to use scripts from `bin` directory, and since you did it in line 32 for rails so you could use it here as well:

```suggestion
        run: bin/bundle exec rspec
      - name: Security audit dependencies
        run: bin/bundle exec bundler-audit --update
      - name: Security audit application code
        run: bin/bundle exec brakeman -q -w2
      - name: Lint Ruby files
        run: bin/bundle exec rubocop --parallel
```